src/app/user/user-location/user-location.component.html

```diff
@@ -0,0 +1,33 @@
+<script>
```

Comment:
Maybe do it inside ts?

Reply:
ill, it was just for checking the keys i had. now it is not wirking so just merge it as it is